docs/DeveloperGuide.md

```diff
@@ -320,26 +321,26 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | -------- | ------------------------------------------ | ------------------------------ | ---------------------------------------------------------------------- |
 | `* * *`  | user starting to use the app| be able to add in the contacts of my CCA members | have a better overview of the members in my CCA and access their information easily|
 | `* * *`  | forgetful CCA leader | be able to view who have been added into my contacts | retrieve their information and add the remaining members too |
-| `* * *`  | CCA leader with many tasks to be done in the CCA | be able to add tasks into the application | better manage the tasks |
-| `* * *`  | CCA leader with tasks that are constantly changing in details | be able to edit the details of the tasks accordingly | keep the task details updated |
+| `* * *`  | CCA leader with many tasks to be done | be able to add the tasks of our CCA | better manage the tasks |
+| `* * *`  | CCA leader whose tasks' details are needed to be updated constantly | be able to edit the details of the tasks accordingly | keep the tasks' details updated |
```

Comment:
I think it should be "CCA leader whose tasks' details need to be updated constantly" ?

Reply:
okay